domains/transfer-agreements/TransferAgreements.API/API/Controllers/TransferAgreementsController.cs

```diff
@@ -37,4 +40,24 @@ public async Task<ActionResult> Create([FromBody] CreateTransferAgreement reques
 
         return Created($"api/transfer-agreements/{result.Id}", result);
     }
+
+    [ProducesResponseType(200)]
```

Comment:
Please indicate the type returned, e.g. [ProducesResponseType(typeof(TransferAgreementsResponse), 200)]

Reply:
Done